pkg/simple_cache/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: simple_cache
+description: A simple cache abstraction wrapping in-memory, redis, memcached.
+publish_to: None # TODO: consider splitting this package into:
+                 # simple_cache             // Cache wrapper
+                 # simple_cache_provider    // CacheProvider interface + Inmemory
+                 # simple_cache_redis       // CacheProvider implementation for redis
+environment:
+  sdk: '>=2.0.0 <3.0.0'
+
+dependencies:
+  logging: '^0.11.3'
+  identity_codec:
+    path: ../identity_codec
+  retry:
+    path: ../retry
+  dartis:
+    git:
```

Comment:
I think this is a high-risk dependency even if you are in control of that repository, going outside of the usual review cycle. I'd rather have a copy hosted here too.

Reply:
there is work in progress to have it published. I would want that before we land this PR :)
Otherwise, we should vendor it into `third_party/` per policy.